vavr/src/main/java/io/vavr/CheckedConsumer.java

```diff
@@ -29,6 +32,29 @@
 @FunctionalInterface
 public interface CheckedConsumer<T> {
 
+    /**
+     * Creates a {@code CheckedConsumer}.
+     *
+     * <pre>{@code
+     * final CheckedConsumer<Value> checkedConsumer = CheckedConsumer.of(Value::stdout);
+     * final Consumer<Value> consumer = checkedConsumer.unchecked();
+     *
+     * // prints "Hi" on the console
+     * consumer.accept(CharSeq.of("Hi!"));
+     *
+     * // throws
+     * consumer.accept(null);
+     * }</pre>
+     *
+     * @param methodReference (typically) a method reference, e.g. {@code Type::method}
+     * @param <T> type of values that are accepted by the consumer
+     * @return a new {@code CheckedConsumer}
+     * @see CheckedFunction1#of(CheckedFunction1)
+     */
+    static <T> CheckedConsumer<T> of(CheckedConsumer<T> methodReference) {
+        return methodReference;
+    }
```

Comment:
We also need factory methods in order to be able to take advantage of the new unchecked() method.